apps/linode-marketplace-code-server/roles/code-server/tasks/main.yml

```
@@ -0,0 +1,81 @@
+---
+- name: install required packages
+  apt:
+    pkg:
+      - nginx
+      - curl
+      - git
+    state: latest
+    update_cache: true
+
+# code-server
+- name: download code-server .deb package
+  get_url:
+    url: "{{ code_download_link }}"
+    dest: "/tmp/code-server_{{ code_server_version }}_amd64.deb"
+
+- name: install code-server .deb package
+  apt:
+    deb: "/tmp/code-server_{{ code_server_version }}_amd64.deb"
+    state: present
```

```suggestion
- name: download code-server .deb package
  get_url:
    url: "{{ code_download_link }}"
    dest: "/tmp/code-server_{{ code_server_version }}_amd64.deb"
  register: code_server_deb

- name: install code-server .deb package
  apt:
    deb: "{{ code_server_deb.dst }}"
    state: present
```

We can register a variable for get_url and use the dst return value.